hnn_core/viz.py

```diff
@@ -444,6 +453,10 @@ def plot_spectrogram(dpl, *, fmin, fmax, winlen=None, tmin=None, tmax=None,
         The layer to plot. Can be one of 'agg', 'L2', and 'L5'
     ax : instance of matplotlib figure | None
         The matplotlib axis.
+    units : str | None
```

Comment:
oh but here this "unit" is not used in the end. I feel it's a bit of a surprising behavior. Someone may pass the full units of power.